UIcode/README.md

```diff
@@ -0,0 +1,25 @@
+# React File Upload
+
+> This is a full stack React-Express file uploader but could easily be modified to work with any back-end including cloud storage
+
+## Quick Start
+
+```bash
+# Install dependencies server/client
+npm install
+cd client
+npm install
+
+# Create .env.development file as follows
+touch .env.development
+
+# set API url i.e. http://localhost:4080
+echo "REACT_APP_API_URL=http://localhost:4080" >> .env.development
```

Comment:
I am confused by this instruction to configure the API on port 4080, but the server is started below on port 3000. Can you please document this better, explaining the difference between the two ports if they are indeed different ports?

Reply:
@alaaltoros ?

Reply:
This is the URL of the backend API, I added more description to this file.